tests/test_models.py

```diff
@@ -41,6 +46,23 @@ def test_parse_should_parse_node_id_attributes(self, pattern_config):
         )
     ))
     def test_parse_with_class_name(self, pattern_config, nodeid, class_name):
-        node = models.Node.parse(nodeid, pattern_config)
+        node = Node.parse(nodeid, pattern_config)
 
         assert node.class_name == class_name
+
+    def test_repr_should_return_a_string_representation_of_itself(self, node):
+        from_repr = eval(repr(node))
+
+        assert from_repr.title == node.title
```

Comment:
This could be tested be using `__eq__`, but since it would be implemented to be used only in tests, it's not necessary.